stable/pipelines/templates/pipelines-statefulset.yaml

```diff
@@ -64,11 +64,9 @@ spec:
             echo "Waiting for Vault to come up...";
             {{- if .Values.vault.enabled }}
             until nc -z -w 2 {{ include "pipelines.vault.name" . }} {{ .Values.vault.service.port }} && echo Vault ok; do
-            {{- else }}
-            until nc -z -w 2 {{ tpl .Values.global.vault.host . }} {{ .Values.global.vault.port }} && echo Vault ok; do
-            {{- end }}
               sleep 2;
             done;
+            {{- end }}
```

Comment:
please add external vault url uptime check

Reply:
Added the check for external vault availability.